packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -102,7 +102,7 @@ Example usage of the `nimble-rich-text-editor` in the application layer is as fo
 
 _Props/Attrs_
 
--   `markdownValue` - is an accessor used to get and set the markdown value.
+-   `markdown` - is an accessor used to get and set the markdown value.
```

Comment:
This makes sense. 👍

The native input has the concept of [`value`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#value) (the raw string value), [`valueAsNumber`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#valueasnumber), and [`valueAsDate`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#valueasdate).

If we had a concept of a "raw" `value` then I can imagine us doing `valueAsMarkdown`, etc. But I don't think we will ever expose a `value` property for the "raw" value so the `valueAs*` pattern isn't really worth following. So this change seems reasonable to me 👍.